site/content/docs/11-deploying.md

```diff
@@ -5,18 +5,15 @@ title: Deployment
 Sapper apps run anywhere that supports Node 8 or higher.
 
 
-### Deploying to Now
-
-> This section relates to Now 1, not Now 2
-
-We can very easily deploy our apps to [Now][]:
+### Deploying to Vercel
+We can very easily deploy our apps to [Vercel][]):
 
 ```bash
-npm install -g now
-now
+npm install -g vercel
+vercel
 ```
 
-This will upload the source code to Now, whereupon it will do `npm run build` and `npm start` and give you a URL for the deployed app.
+This will upload the source code to Vercel, whereupon it will do `npm run build` and `npm start` and give you a URL for the deployed app.
```

Comment:
This is not accurate: it will run `sapper export`

Reply:
The `vercel` tool can't know about the `sapper` command, so I don't think that's right. Anyway, I don't think it will work to just run `vercel` alone and we'll need to use the other set of proposed instructions that mention the plugin: https://github.com/sveltejs/sapper/pull/1199

Reply:
Vercel runs `now-build` if it exists (vercel-build?), and then serves up the public directory. This `npm run build` command may or may not run `sapper export` depending on how your `package.json` is configured

I don't think it will run `npm start`. It's focussed on static apps, you'd have to configure a it as a `@now/node` application in the `now.json` configuration in order for it to run start.

Reply:
@benmccann `vercel` is actually sapper aware: https://github.com/zeit/now/blob/8a68211cad8d2afae02ac1b85184e07d8911d47b/packages/now-static-build/src/frameworks.ts#L420
It's just too bad that they didn't go for the full SSR approach. Maybe now that this repo is getting stable, we should do a PR.

Reply:
Oh, that's cool! I noticed that file is in a directory called `now-static-build`, so I'm not sure if they'd accept a PR or not. You might be able to raise an issue to ask if there's a good place to contribute it though